packages/pieces/google-drive/src/lib/action/download-file.ts

```diff
@@ -0,0 +1,87 @@
+import { googleDriveAuth } from '../..';
+import { createAction, Property } from '@activepieces/pieces-framework';
+import { extension, lookup } from 'mime-types';
+
+export const downloadFile = createAction({
+  auth: googleDriveAuth,
+  name: 'download_ggdrive_file',
+  displayName: 'Download file',
+  description: 'Download a selected file from google drive file',
+  props: {
+    fileId: Property.ShortText({
+      displayName: 'File ID',
+      description: 'The ID of the file to download',
+      required: true,
+    }),
+    mimeType: Property.ShortText({
+      displayName: 'MIME type',
+      description: 'The MIME type of the file to download',
+      required: true,
+    }),
+    fileName: Property.ShortText({
+      displayName: 'File name',
+      description: 'The indicative name of the file to download',
```

Comment:
I think destination file name and the description would be that not the file name to download but rather what name to save it with.